proto/scene.proto

```diff
@@ -162,27 +284,45 @@ message Camera {
   int32 id = 1;
   Transform transform = 2;
   oneof camera {
-    // @oneof 2 camera
-    PerspectiveCamera perspective = 3;
+    PerspectiveCamera perspective = 3; // @oneof 2 camera
     // @oneof_item
     // @optional
     OrthographicCamera orthographic = 4;
   }
 }
 
-// Representation of material
+/*
+Representation of material
+For more detailed explanation follow <a href="https://www.scratchapixel.com/lessons/3d-basic-rendering/phong-shader-BRDF">link</a>
+or this <a href="https://pbr-book.org/3ed-2018/Reflection_Models">link</a>
+*/
 message Material {
   string id = 1;
   oneof material {
-    // @oneof 2 material
-    SolidMaterial solid = 2;
-    // @oneof_item
-    MirrorMaterial glass = 3;
+    LambertReflectionMaterial lambertReflection = 2; // @oneof 2 material
```

Comment:
Please use snake_case. Protobuf will translate it to a right style for each language (i.e. camelCase for C#)